src/main/java/entities/validity_checkers/PunctuationValidityCheckerRegular.java

```diff
@@ -0,0 +1,33 @@
+package entities.validity_checkers;
+
+/**
+ * Punctuation validity checker for the regular game mode
+ */
+public class PunctuationValidityCheckerRegular implements PunctuationValidityChecker {
+
+    private static final int PUNC_LENGTH = 3;
+
+    /**
+     * Verifies punctuation is valid based on game mode regular criteria
```

Comment:
Maybe be more specific in the doc string regarding what valid punctuation combinations look like. This punctuation checker may be reused across multiple game modes in the future and not necessarily just GameRegular.